Translations update from Weblate

weblate opened the pull request:
Translations update from [Weblate](https://hosted.weblate.org/projects/TheAlgorithms/common/) for The Algorithms/Common.



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widgets/TheAlgorithms/-/common/horizontal-auto.svg)

AnshulMalik commented on css/style.css:
```diff
@@ -23,9 +23,9 @@ a {
 }
 
-a:not(.github):hover {
+/* a:not(.github):hover {
```
if we don't need it we can just remove it.

Ramanpreet6262 replied:
Ok I'll remove it

AnshulMalik commented on css/style.css:
```diff
@@ -77,3 +76,7 @@ div.grid-item {
 }
+
+.language-name:hover{
+  border-bottom: 3px solid #1b6568;
```
Two points here.
1. I think we are missing the little transition effect `transition-duration: 100ms;` here
2. New line.

Ramanpreet6262 replied:
Transition effect I'll add.
What's new line?

AnshulMalik commented on css/style.css:
```diff
@@ -77,3 +76,7 @@ div.grid-item {
+
+.language-name:hover{
+  border-bottom: 3px solid #1b6568;
+}
```
The new line is just a lint thing, should have an empty line at the end of a file.

Ramanpreet6262 replied:
ok

maltejur committed:
Translated using Weblate (German)

Currently translated at 93.9% (31 of 33 strings)

Translation: The Algorithms/Common
Translate-URL: https://hosted.weblate.org/projects/TheAlgorithms/common/de/

maltejur committed:
Translated using Weblate (German)

Currently translated at 100.0% (33 of 33 strings)

Translation: The Algorithms/Common
Translate-URL: https://hosted.weblate.org/projects/TheAlgorithms/common/de/
